src/ckms.rs

```diff
@@ -50,11 +51,13 @@ pub struct CKMS<T: Copy> {
     samples: Vec<Entry<T>>,
 
     sum: Option<T>,
+    cma: Option<f64>,
     last_in: Option<T>,
 }
 
 impl<T> AddAssign for CKMS<T>
-    where T: Copy + Add<Output = T> + PartialOrd + Debug
+    where T: Copy + Add<Output = T> + Sub<Output = T> + Div<Output = T> +
```

Comment:
Sure! `CMKS` is generic on its numeric type `T`. To signal that we need some operations available we gotta add type constraints, which is what's going on here. For instance

    Sub<Output = T>

says that `T` must support the subtraction operation and that `- :: T -> T -> T`.